### DISCUSSION TITLE
Existing BS Roster Editor - things we DO like

### DISCUSSION THREAD

**Discussion by amis92** [BSData/phalanx#14] [Ideas]
Please add any and all the *good* features of the existing BS roster editor.

**Comment by AlexMorman** [comment]
I like being able to save/favorite a unit configuration

**Comment by AlexMorman** [comment]
I like being able to write notes for individual units, forces, or armies

**Comment by jlueck6** [comment]
I think the overall app layout is pretty great; I think it's difficult to argue against a list-centric layout when the app's main job is building lists. (Mobile, android)
